cmd/lakectl/cmd/store/factory.go

```diff
@@ -45,7 +47,7 @@ func Walk(ctx context.Context, storageURI string, walkFn func(e ObjectStoreEntry
 	}
 	switch uri.Scheme {
 	case "s3":
-		svc, err := GetS3Client()
+		svc, err := GetS3Client(*aws.NewConfig())
```

Comment:
Just question - what is the difference between the default we used to have and passing the default new configuration?

Reply:
No difference in this context, which is exactly what I needed                                                                       .
On the other hand, from the test, I can force certain params, such as the region, which is required in order to access the `nessie-system-testing-data` bucket